src/integrationTest/java/com/mongodb/kafka/connect/MongoSourceConnectorTest.java

```diff
@@ -361,6 +362,40 @@ void testSourceLoadsDataFromCollectionCopyExisting() {
     }
   }
 
+  @Test
+  @DisplayName("Ensure source loads data from collection with copy existing data - outputting bson")
+  void testSourceLoadsDataFromCollectionCopyExistingBson() {
+    MongoCollection<Document> coll = getAndCreateCollection();
+
+    insertMany(rangeClosed(1, 50), coll);
+
+    Properties sourceProperties = new Properties();
+    sourceProperties.put(MongoSourceConfig.DATABASE_CONFIG, coll.getNamespace().getDatabaseName());
+    sourceProperties.put(
+        MongoSourceConfig.COLLECTION_CONFIG, coll.getNamespace().getCollectionName());
+    sourceProperties.put(MongoSourceConfig.COPY_EXISTING_CONFIG, "true");
+    sourceProperties.put(MongoSourceConfig.OUTPUT_FORMAT_KEY_CONFIG, OutputFormat.BSON.name());
+    sourceProperties.put(MongoSourceConfig.OUTPUT_FORMAT_VALUE_CONFIG, OutputFormat.BSON.name());
+    sourceProperties.put("key.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
```

Comment:
Converters are an abstraction above the SourceRecord. Their job is to convert the data in the SourceRecord to the type put on the topic.  By default the string converter is used (hence all the raw json string tests working as expected).